devops/staging/values.yaml

```
@@ -122,14 +122,17 @@ iam-cache-server-helm:
   opsValues:
     DB_NAME: ics
     NODE_ENV: production
-    IPFS_URL: https://ipfs.infura.io:5001/api/v0/
     ENS_URL: https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/
     UNIVERSAL_RESOLVER_URL: https://dev.uniresolver.io/10/identifiers/
     NATS_VERSION: 2.1.8
     NATS_CLIENTS_URL: iam-cache-server-nats.ics.svc.cluster.local:4222
     LOGS_DIRECTORY: /app/logs
     NESTJS_PORT: 3000
     SENTRY_ENV: staging
+    IPFS_CLIENT_PORT: 5001
+    IPFS_CLIENT_PROJECT_SECRET: 9506198a91243f03417aaed60e5eb654
+    IPFS_CLIENT_PROJECT_ID: 2DBFBP5zsjfImw1wBPn1Vx08Yd5
```

I created a new project for staging
```suggestion
    IPFS_CLIENT_PROJECT_SECRET: 6ff4f07ba7643a75d743461b288104c0
    IPFS_CLIENT_PROJECT_ID: 2DDGgPm6nTDVR6HS7o0wjY5ju5j
```